ors-api/src/main/resources/log4j.properties

```diff
@@ -4,7 +4,7 @@ log4j.rootLogger=DEBUG, A1
 # A1 is set to be a ConsoleAppender.
 log4j.appender.A1=org.apache.log4j.ConsoleAppender
 log4j.appender.A1.layout=org.apache.log4j.PatternLayout
-log4j.appender.A1.layout.ConversionPattern=%d{dd MMM HH:mm:ss} %p [%c{2}] - %m%n
+#log4j.appender.A1.layout.ConversionPattern=%d{dd MMM HH:mm:ss} %p [%c{2}] - %m%n
```

Comment:
Why not remove?

Reply:
Forgot to uncomment, I was just trying something. Fixed.